### PR TITLE
[zh] Update persistent-volumes.md

### PR DESCRIPTION
**Pull request by fenggw-fnst**
Signed-off-by: Guangwen Feng <fenggw-fnst@cn.fujitsu.com>

en PR https://github.com/kubernetes/website/pull/30517/files#diff-8a8f3a263c8539f0d3ef13e56424170aa3564788e33efbc956c9339da1765a9a
/assign @tengqm
PTAL, thanks.

### CODE REVIEW

**Comment by tengqm** [comment]
/hold
This file is out of sync. Check the diffs using the following command on the **main** branch:

```shell
./scripts/lsync.sh content/zh/docs/concepts/storage/persistent-volumes.md
```

**Comment by tengqm** [comment]
@fenggw-fnst Are you still interested in working on this?

**Comment by fenggw-fnst** [comment]
Ah sorry, I've been busy with other work recently.

> Are you still interested in working on this?

Yes, I will update it within this month.

**Comment by fenggw-fnst** [comment]
@tengqm Sorry for the delay and thanks for your patience, I have updated it. PTAL.

File: content/zh/docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -605,6 +631,53 @@ If expanding underlying storage fails, the cluster administrator can manually re
...
 
+{{% /tab %}}
+{{% tab name="通过请求扩展为更小尺寸" %}}
+{{% feature-state for_k8s_version="v1.23" state="alpha" %}}
+
+<!--
+Recovery from failing PVC expansion by users is available as an alpha feature since Kubernetes 1.23. The `RecoverVolumeExpansionFailure` feature must be enabled for this feature to work. Refer to the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) documentation for more information.
+-->
+{{< note >}}
+Kubernetes 从 1.23 版本开始将通过用户从失败的 PVC 扩展中进行恢复这一能力作为
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/persistent-volumes.md:642]:
```suggestion
Kubernetes 从 1.23 版本开始将允许用户恢复失败的 PVC 扩展这一能力作为
```

File: content/zh/docs/concepts/storage/persistent-volumes.md [hunk 2]
```diff
@@ -605,6 +631,53 @@ If expanding underlying storage fails, the cluster administrator can manually re
...
+smaller size than the previously requested value. To request a new expansion attempt with a
+smaller proposed size, edit `.spec.resources` for that PVC and choose a value that is less than the
+value you previously tried.
+This is useful if expansion to a higher value did not succeed because of capacity constraint.
+If that has happened, or you suspect that it might have, you can retry expansion by specifying a
+size that is within the capacity limits of underlying storage provider. You can monitor status of resize operation by watching `.status.resizeStatus` and events on the PVC.
+-->
+如果集群中的特性门控 `ExpandPersistentVolumes` 和 `RecoverVolumeExpansionFailure`
+都已启用，在 PVC 的扩展发生失败时，您可以使用比先前请求的值更小的大小来重试扩展。
+要使用一个更小的大小尝试请求新的扩展，请编辑该 PVC 的 `.spec.resources` 并选择
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/persistent-volumes.md:660]:
```suggestion
要使用一个更小的尺寸尝试请求新的扩展，请编辑该 PVC 的 `.spec.resources` 并选择
```

File: content/zh/docs/concepts/storage/persistent-volumes.md [hunk 3]
```diff
@@ -605,6 +631,53 @@ If expanding underlying storage fails, the cluster administrator can manually re
...
+一个比您之前所尝试的值更小的值。
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/persistent-volumes.md:661]:
```suggestion
一个比你之前所尝试的值更小的值。
```

File: content/zh/docs/concepts/storage/persistent-volumes.md [hunk 4]
```diff
@@ -605,6 +631,53 @@ If expanding underlying storage fails, the cluster administrator can manually re
...
+如果由于容量限制而无法成功扩展至更高的值，这将很有用。
+如果发生了这种情况，或者您怀疑它可能发生了，您可以通过指定一个在底层存储供应容量
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/persistent-volumes.md:663]:
```suggestion
如果发生了这种情况，或者你怀疑可能发生了这种情况，你可以通过指定一个在底层存储供应容量
```

File: content/zh/docs/concepts/storage/persistent-volumes.md [hunk 5]
```diff
@@ -605,6 +631,53 @@ If expanding underlying storage fails, the cluster administrator can manually re
...
+限制内的大小来重试扩展。您可以通过查看 `.status.resizeStatus` 以及 PVC 上的事件
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/persistent-volumes.md:664]:
```suggestion
限制内的尺寸来重试扩展。你可以通过查看 `.status.resizeStatus` 以及 PVC 上的事件
```

File: content/zh/docs/concepts/storage/persistent-volumes.md [hunk 6]
```diff
@@ -605,6 +631,53 @@ If expanding underlying storage fails, the cluster administrator can manually re
...
+来监控调整大小操作的状态。
+
+<!--
+Note that,
+although you can a specify a lower amount of storage than what was requested previously,
+the new value must still be higher than `.status.capacity`.
+Kubernetes does not support shrinking a PVC to less than its current size.
+-->
+请注意，
+尽管您可以指定比之前的请求更低的存储量，新值必须仍然高于 `.status.capacity`。
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/persistent-volumes.md:674]:
```suggestion
尽管你可以指定比之前的请求更低的存储量，新值必须仍然高于 `.status.capacity`。
```

File: content/zh/docs/concepts/storage/persistent-volumes.md [hunk 7]
```diff
@@ -605,6 +631,53 @@ If expanding underlying storage fails, the cluster administrator can manually re
...
+Kubernetes 不支持将 PVC 缩小到小于其当前的大小。
```
Comment (tengqm) [inline content/zh/docs/concepts/storage/persistent-volumes.md:675]:
```suggestion
Kubernetes 不支持将 PVC 缩小到小于其当前的尺寸。
```

**Comment by fenggw-fnst** [comment]
Thanks for the review, done.

**Pull request merged**